map-sdk/js/api/set-view-state/index.html

```diff
@@ -0,0 +1,140 @@
+<body style="margin: 0">
+<div 
+  class="codepen" 
+  data-prefill='{
+    "title": "Unfolded Map SDK - setViewState example",
+    "description": "",
+    "html_classes": ["loading", "no-js"],
+    "head": "&lt;meta name=&#x27;viewport&#x27; content=&#x27;width=device-width, initial-scale=1&#x27;&gt;",
+    "stylesheets": [
+      "https://cdn.unfolded.ai/statics/fonts/unfolded-fonts.css",
+      "https://cdn.unfolded.ai/sdk/examples/style.css"
+    ],
+    "scripts": [
+      "https://unpkg.com/@unfolded/map-sdk@0.2.0/dist/unfolded-sdk.min.js"
+    ]
+  }'
+  data-height="800" 
+  data-theme-id="1"
+  data-default-tab="result" 
+  data-editable="true"
+>
+<pre data-lang="html">
+&lt;div class="unfolded"&gt;
```

Comment:
Do we need all these escapes `&lt`?

Reply:
That was a leftover from CodePen. Cleaned it up now.